modules/01-mods.js

```diff
@@ -50,6 +53,39 @@ async function slashModBan(interaction) {
   await c.ban(interaction, target, reason, days);
 }
 
+async function slashModFilter(interaction) {
+  await interaction.deferReply({ ephemeral: true });
+  const word = interaction.options.getString("word").toLowerCase().trim();
+  const user = interaction.member;
+  const modLogs = interaction.guild.channels.cache.get(sf.channels.modlogs);
+  const filtered = pf.scan(word);
+  const apply = interaction.options.getBoolean("apply") ?? true;
+  if (!user.roles.cache.has(sf.roles.management) && !user.roles.cache.has(sf.roles.manager) && !sf.adminId.includes(user.id)) {
+    interaction.editReply("This command is for management only.");
```

Comment:
Let's make this more specific to match the actual permissions being checked.
```suggestion
    interaction.editReply("This command is for Management, Discord Manager, and Bot Admins only.");
```